merkletree/version.go

```diff
@@ -1,5 +1,7 @@
 package merkletree
 
 const (
-	Version = "0.1"
+	// Version indicates the current version of the library.
+	// This is matched with our release version on Github.
+	Version = "0.1.0"
```

Comment:
Hmm, `0.1.0` is definitely the library version, I'm not sure how to number our protocol version. I'd say that the version in the USENIX paper is the second iteration, and after working with you this summer, we've reiterated over it, so I'd say we're on `v3`.